src/WebRTCSession.cpp

```diff
@@ -21,6 +21,9 @@ WebRTCSession::WebRTCSession()
 {
         qRegisterMetaType<WebRTCSession::State>();
         connect(this, &WebRTCSession::stateChanged, this, &WebRTCSession::setState);
+#if GST_CHECK_VERSION(1, 18, 0)
```

Comment:
```suggestion
#if defined (GSTREAMER_AVAILABLE) && GST_CHECK_VERSION(1, 18, 0)
```

Reply:
Hm, seems like that was not the way to fix it, sorry about that. Let me think about how to actually do this properly, maybe the check needs to be wrapped inside an #ifdef because of function macros? :D

Reply:
Thanks but honestly there's no real reason not to just init gstreamer at start up under 1.16 too. It's just not absolutely required like it is under 1.18. I'll confirm and then just remove the test.

Reply:
Nuked it from orbit. Sorry for the hassle.

Reply:
No worries, I think this is actually great, it removes some complexity of those ifdefs :3